www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+     and for any reason, including for violations of these Terms. We will make
+reasonable efforts to notify you if we suspend or terminate your account.
+
+  1. Upon termination of your account, Gratipay shall make reasonable efforts
+     to remove your content from the Service, except where doing so would
+adversely affect the experience of other Participants, and to cease all use of
+your content. However, you acknowledge and agree that: (a) removed content may
+persist in caches or backups for a reasonable period of time and (b) copies of
+or references to your content may not be entirely removed, for example because
+it has been copied or referenced by other Participants.
+
+  1. In the event of termination, sections 5, 14, 15, 16, and 17 of these Terms
+     shall remain in effect, as will any terms that by their nature extend
+beyond termination.
+
+1. Disclaimer of Warranties, Limitations of Remedies
+
+  1. Gratipay makes no representations or warranties of any kind with respect
+     to the Service, including any representation or warranty that the use of
+the Service will (a) be timely, uninterrupted or error-free or operate in
+combination with any other hardware, software, system, or data, (b) meet your
+requirements or expectations, (c) be free from errors or that defects will be
+corrected, or (d) be free of viruses or other harmful components. Gratipay also
+makes no representations or warranties of any kind with respect to any content.
+No advice or information, whether oral or written, obtained from Gratipay or
+through the Service, will create any warranty not expressly made herein.
+
+  1. Your access to and use of the Service or any content is at your own risk.
+     YOU UNDERSTAND AND AGREE THAT THE SERVICE IS PROVIDED TO YOU ON AN "AS IS"
+AND "AS AVAILABLE" BASIS. WITHOUT LIMITING THE FOREGOING, TO THE FULL EXTENT
+PERMITTED BY LAW, GRATIPAY DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, OR NON-INFRINGEMENT.
+
+1. Limitation of Liability
+
+  1. TO THE MAXIMUM EXTENT PERMITTED BY APPLICABLE LAW, GRATIPAY SHALL NOT BE
+     LIABLE FOR ANY INDIRECT, INCIDENTAL, SPECIAL, CONSEQUENTIAL OR PUNITIVE
+DAMAGES, OR ANY LOSS OF PROFITS OR REVENUES, WHETHER INCURRED DIRECTLY OR
+INDIRECTLY, OR ANY LOSS OF DATA, USE, GOOD-WILL, OR OTHER INTANGIBLE LOSSES,
+RESULTING FROM (i) YOUR ACCESS TO OR USE OF OR INABILITY TO ACCESS OR USE THE
+SERVICE; (ii) ANY CONDUCT OR CONTENT OF ANY THIRD PARTY ON THE SERVICE,
+INCLUDING WITHOUT LIMITATION, ANY DEFAMATORY, OFFENSIVE OR ILLEGAL CONDUCT OF
+OTHER PARTICIPANTS OR THIRD PARTIES; (iii) ANY CONTENT OBTAINED FROM THE
+SERVICE; OR (iv) UNAUTHORIZED ACCESS, USE OR ALTERATION OF YOUR TRANSMISSIONS
+OR CONTENT.  IN NO EVENT SHALL THE AGGREGATE LIABILITY OF GRATIPAY OR ITS
+AFFILIATES EXCEED THE GREATER OF ONE HUNDRED U.S. DOLLARS (U.S. $100.00) OR THE
+AMOUNT YOU PAID GRATIPAY, IF ANY, IN THE PAST SIX MONTHS FOR THE SERVICE GIVING
```

Comment:
From @copiesofcopies:

>YES WE SHOULD LEAVE THE LIABILITY LIMITATION AS IS because people will sue over anything, even voluntary payments they made.

E.g., http://en.wikipedia.org/wiki/Jonathan_Lee_Riches as @webmaven mentions at https://github.com/gratipay/gratipay.com/pull/3408#discussion_r30290466. :-)